dsfhub/resource_log_aggregator.go

```diff
@@ -816,6 +816,14 @@ func resourceLogAggregatorUpdate(d *schema.ResourceData, m interface{}) error {
 	serverType := d.Get("server_type").(string)
 	createResource(&logAggregator, serverType, d)
 
+	log.Printf("[DEBUG] AuditPullEnabled value in logAggregator is: '%v'", logAggregator.Data.AssetData.AuditPullEnabled)
```

Comment:
Would this make more sense to print _after_ we set the value? Or perhaps, before and after?

Reply:
Yea I can put one after too. Good suggestion!